src/components/about/About.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import "./about.scss"
+
+const About = props => {
+  return (
+    <div className="outer-container">
+      <div className="about-container">
+        <h1>
+          <span id="about-us-header">ABOUT US</span>
```

Comment:
This `span` looks like it might be unnecessary - at least right now. I'd recommend moving your id to the `h1` and removing the `span` if that works just as well.

Reply:
Removed `span`, moved `id` to `H1`